src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
Instead of importing the whole React, you can import only the strictmode
```suggestion
import { StrictMode } from 'react'
```